rs/crypto/internal/crypto_lib/multi_sig/bls12_381/src/api.rs

```diff
@@ -115,6 +115,7 @@ pub fn verify_individual(
             algorithm: AlgorithmId::MultiBls12_381,
             public_key_bytes: public_key_bytes.0.to_vec(),
             sig_bytes: signature_bytes.0.to_vec(),
+            msg_hash: Some(message.to_vec()),
```

Comment:
IIUC, `message` is not a hash here, but the output of `Signable::as_signed_bytes` which uses `SignedBytesWithoutDomainSeparator::as_signed_bytes_without_domain_separator`, which depends on the actual struct that is signed.

Looking at `pub trait Crypto`, multi signatures are done for `NotarizationContent` and `FinalizationContent`, and both implement `SignedBytesWithoutDomainSeparator` by serializing self with CBOR.

Same for `verify_combined`.